modules/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureBlobStore.java

```diff
@@ -169,6 +170,7 @@ && isPutBlockListRequest(httpMethod, url) == false,
                     return;
                 }
             }
+            logger.debug("Request did not match any of the tracked patterns: {} {}", method, url);
```

Comment:
This might be useful if we think the stats are not being tracked correctly. Currently we silently ignore requests we can't identify.